sdk/nodejs/stackReference.ts

```diff
@@ -31,6 +31,11 @@ export class StackReference extends CustomResource {
      */
     public readonly outputs: Output<{[name: string]: any}>;
 
+    /**
+     * The names of any stack outputs which contain secrets.
+     */
+    public readonly secretOutputs: Output<string[]>;
```

Comment:
fwiw, this naming confused me.  because you have `outputs` which is a map from string to something.  But then `secretOutputs` which is not a map.  I'd call this `secretOutputNames` to be clearer.

Reply:
Renamed.